### ISSUE TITLE
CSS transition for menu

### ISSUE THREAD

**Issue by wickkidd** [riophae/vue-treeselect#85]
I need to apply a CSS transition to the menu (.vue-treeselect__menu), however, because v-if is used, the element enters and leaves the DOM so that's currently impossible. If it were converted to a v-show or a slot was created for it (this might be more difficult because the v-if would have to be implemented by the user) to allow me to wrap it in a vue transition component, this would work.

https://github.com/riophae/vue-treeselect/blob/9252da8188a724e66a2a6558cd93c041bb5fd9c5/src/components/Treeselect.vue#L34

**Comment by riophae** [comment]
Hi! I just added a [transition wrapper](https://vuejs.org/v2/guide/transitions.html#Transitioning-Single-Elements-Components) to the menu. This enables you to customize the animation using CSS:

```less
.vue-treeselect__menu--transition {
  &-enter-active,
  &-leave-active {
    transition: transform 100ms;
    transform-origin: top center;
  }

  &-enter,
  &-leave-to {
    transform: scaleY(0);
  }
}
```

**Comment by riophae** [comment]
Landed in v0.0.27.

**Comment by wickkidd** [comment]
Wow that was fast! :)

Works well. Thanks.
